egs/wsj/s5/steps/nnet3/report/generate_plots.py

```diff
@@ -654,9 +653,13 @@ def main():
     else:
         output_nodes.append(('output', 'linear'))
 
-    generate_plots(args.exp_dir, args.output_dir, output_nodes,
-                   comparison_dir=args.comparison_dir,
-                   start_iter=args.start_iter)
+    if len(args.exp_dir) == 1: 
+      generate_plots(args.exp_dir[0], args.output_dir, output_nodes,
```

Comment:
Remember to specify in the help message that the first directory would be treated as the main experiment directory. It would be used to determine the components will have non-linearity and parameter stats plotted.